src/components/Nav/TeamSideNavButton.vue

```diff
@@ -0,0 +1,54 @@
+<script>
+import { mapGetters, mapMutations } from 'vuex'
+
+export default {
+  data() {
+    return {}
+  },
+  computed: {
+    ...mapGetters('sideNav', ['isOpen']),
+    ...mapGetters('tenant', ['tenant']),
+    ...mapGetters('user', ['invitations']),
+    size() {
+      return this.$vuetify.breakpoint.mdAndUp ? (this.isOpen ? 300 : 200) : 150
+    }
+  },
+  methods: {
+    ...mapMutations('sideNav', ['toggle'])
+  }
+}
+</script>
+
+<template>
+  <v-btn
+    class="text-subtitle-1 text-capitalize mx-1 font-weight-medium d-flex justify-space-between"
+    dark
+    large
+    depressed
+    :width="size"
+    style="
+      background-color: rgba(255, 255, 255, 0.1);
+      transition: width 200ms;"
+    title="Open the team sidebar"
```

Comment:
I'm not sure the team reference is true anymore/fully reflects what the sidenav shows 
```suggestion
    title="Open the sidebar"
```